web/client/plugins/Identify.jsx

```diff
@@ -164,6 +154,11 @@ const identifyDefaultProps = defaultProps({
  * @prop cfg.disableCenterToMarker {bool} disable zoom to marker action
  * @prop cfg.zIndex {number} component z index order
  * @prop cfg.showInMapPopup {boolean} if true show the identify as popup
+ * @prop cfg.enabled {boolean} // enabled by default
```

Comment:
Correct me if I'm wrong. I think you encountered issues using this variable to set up configuration. This is the reason of a lot of configuration checks, here and there, and attempts to workaround, with this variable.

The problem is that enabled is **always** overridden by the selector in connect. Same for configuration and other properties.
So there is no way, at this stage, to get the enabled value from cfg. (you could if you get it before 'connect', but you can't). Moreover the info `enabled` state need to be always updated in the global state, to sync with the other tools. 

For this reason I suggest you to use defaultConfiguration, that is a new property, not controlled by `connect` as a containers for all the set-up you need to configure on start-up enabled, disableAlwaysOn and configuration. 

Something like
```
defaultConfiguration: {
 enabled: ...
 ....
configuration : { ... } 
}
This allows you to remove all the unnecessary changes (that are wrong or non-sense) in this PR.